cdap-docs/integrations/source/ambari/index.rst

```diff
@@ -0,0 +1,280 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _ambari-configuring:
+
+============================================
+Configuring and Installing CDAP using Ambari
+============================================
+
+
+Overview
+========
+
+.. highlight:: console
+
+This section will familiarize you with CDAP's integration with `Apache Ambari
+<https://ambari.apache.org/>`__, the open source provisioning system for `HDP (Hortonworks
+Data Platform) <http://hortonworks.com/>`__.
+
+You can use `Ambari <https://ambari.apache.org>`__ to integrate CDAP into a Hadoop cluster
+by adding the `CDAP Ambari Services <https://github.com/caskdata/cdap-ambari-service>`__
+to your Ambari server. Once you have restarted your server, you will able to
+use the Ambari UI (Ambari Dashboard) to install, start and manage CDAP on Hadoop clusters.
+
+These instructions cover the steps to integrate CDAP using Apache Ambari:
+
+- **Prerequisites:** Adding the CDAP Service to Ambari.
+- **Adding CDAP:** Adding CDAP to Your Cluster.
+- **Installing CDAP:** Installing CDAP by running the *Add Service* Wizard, and starting CDAP.
+- **Roadmap and Future Features:** Confirming that CDAP was installed and configured successfully.
+
+
+Adding the CDAP Service to Ambari
+=================================
+To install CDAP on a cluster managed by Ambari, we have provided packages for
+RHEL-compatible and Ubuntu systems, which are installed onto the Ambari management server.
+These packages add CDAP to the list of available services which Ambari can install. 
+
+To install the ``cdap-ambari-service`` package, first add the appropriate CDAP repository
+to your system’s package manager by following :ref:`this procedure for installing a Cask
+repository <install-preparing-the-cluster>` on your Ambari server.
```

Comment:
This, by default, will show the 3.0 repository stuff, since it's the 3.0 docs, right?